packages/synthetix-main/contracts/modules/core/MarketManagerModule.sol

```diff
@@ -23,6 +24,7 @@ contract MarketManagerModule is IMarketManagerModule {
     error MarketDepositNotApproved(address market, address from, uint requestedAmount, uint approvedAmount);
 
     function registerMarket(address market) external override returns (uint128 marketId) {
+        FeatureFlag.onlyIfFeatureFlag("market");
```

Comment:
you could probably just rename fhe function to be like `FeatureFlag.ensureEnabled`

Reply:
`"market"` should be a constant, not an inline string

we should probably just make a generla purpose `constants` contract which everything using constants inherits from. I believe solidity is smart enough to optimize out any unused constants so its generally the best option

Reply:
cool, i'll make both these changes 👍🏽